slack.js

```diff
@@ -0,0 +1,266 @@
+//------------------------------------------------------------------------------
+// Bot / Slack related helpers
+//------------------------------------------------------------------------------
+var Botkit = require('botkit');
```

Comment:
alternatively, we could internalize botkit in slackjs. 

either way is fine with me.